### PR TITLE
Bridge_entries_contract | Defines contract for bridge & it's services entries API's.

### PR DESCRIPTION
**Pull request by MounicaSagi**

### CODE REVIEW

File: src/main/resources/static/api.yaml [hunk 1]
```diff
@@ -63,6 +63,109 @@ paths:
...
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/BridgeRegistrationRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/BridgeRegistrationRequest'
+      responses:
+        '200':
+          description: Bridge entry added successfully
```
Comment (ganesanarun) [inline src/main/resources/static/api.yaml]:
```suggestion
          description: Bridge entry added/updated successfully
```

Comment (ganesanarun) [inline src/main/resources/static/api.yaml]:
```suggestion
          description: Bridge entry added/updated successfully
```

File: src/main/resources/static/api.yaml [hunk 2]
```diff
@@ -63,6 +63,109 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /internal/bridges:
+    post:
```
Comment (ganesanarun) [inline src/main/resources/static/api.yaml]:
```suggestion
    put:
```

File: src/main/resources/static/api.yaml [hunk 3]
```diff
@@ -63,6 +63,109 @@ paths:
...
+              * Unhandled exceptions.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /internal/bridges/{bridgeId}/services:
+      post:
```
Comment (ganesanarun) [inline src/main/resources/static/api.yaml]:
```suggestion
      put:
```

File: src/main/resources/static/api.yaml (hunk 2 above)
Comment (MounicaSagi) [reply src/main/resources/static/api.yaml]: done

File: src/main/resources/static/api.yaml (hunk 3 above)
Comment (MounicaSagi) [reply src/main/resources/static/api.yaml]: done

File: src/main/resources/static/api.yaml (hunk 1 above)
Comment (MounicaSagi) [reply src/main/resources/static/api.yaml]: done

Comment (MounicaSagi) [reply src/main/resources/static/api.yaml]: done

**Review by ganesanarun — APPROVED** [review]

**Pull request merged**